administration/src/project-configs/ProjectConfigContext.tsx

```diff
@@ -1,7 +1,8 @@
 import { createContext, ReactNode } from 'react'
 import getProjectConfig, { ProjectConfig } from './getProjectConfig'
 
-const projectConfig = getProjectConfig(window.location.hostname)
+const STAGING_PREFIX = 'staging.'
+const projectConfig = getProjectConfig(window.location.hostname.replace(STAGING_PREFIX, ''))
```

Comment:
I think it would be nicer if we list the different hostnames inside `getProjectConfig`:

```ts
const getProjectConfig = (hostname: string): ProjectConfig => {
  switch (window.localStorage.getItem('project-override') ?? hostname) {
    case 'bayern.ehrenamtskarte.app':
    case 'staging.bayern.ehrenamtskarte.app':
      return bayernConfig
    case 'nuernberg.sozialpass.app':
    case 'staging.nuernberg.sozialpass.app':
      return nuernbergConfig
    default:
      console.debug('Falling back to showcase.')
      return showcaseConfig
  }
}
```